src/stylus/components/_toolbar.styl

```diff
@@ -120,14 +123,6 @@ theme(toolbar, "toolbar")
   &--dense
     toolbar-content(7px)
 
-.toolbar__extension
-  .toolbar__title
-    margin-left: 72px !important
-
-  .tabs__bar
-    align-self: flex-end
```

Comment:
Why was this removed? There's blank space at the bottom of the toolbar now.

Reply:
![image](https://user-images.githubusercontent.com/16421948/34327462-8111e7c6-e919-11e7-80e0-a8e8038a5080.png)
https://codepen.io/anon/pen/baBYLa?editors=1010